File: packages/insomnia-httpsnippet/test/fixtures/output/javascript/jquery/text-plain.js
var settings = {
  "async": true,
  "crossDomain": true,
  "url": "http://mockbin.com/har",
  "method": "POST",
  "headers": {
    "content-type": "text/plain"
  },
  "data": "Hello World"
}

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: packages/insomnia-httpsnippet/test/fixtures/output/python/python3/text-plain.py
import http.client

conn = http.client.HTTPConnection("mockbin.com")

payload = "Hello World"

headers = { 'content-type': "text/plain" }

conn.request("POST", "/har", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))
